templates/confluence.service.erb

```diff
@@ -0,0 +1,13 @@
+[Unit]
+Description=Confluence Team Collaboration Software
+After=network.target
+
+[Service]
+Type=forking
+User=confluence
+PIDFile=<%= scope.lookupvar('confluence::webappdir') %>/work/catalina.pid
```

Comment:
ah, sorry, i think you misunderstood me there ;)

i meant: given the way systemd works (it tracks each service in its own [cgroup](https://www.kernel.org/doc/Documentation/cgroups/cgroups.txt)), do we really need that PIDFile?